README.md

```diff
@@ -20,206 +26,156 @@ There are four mandatory inputs for knodle:
 3. `rule_matches_z`: This matrix shows all applied rules on your dataset. Shape: n_instances x n_rules
 4. `model`: A PyTorch model which can take your provided `model_input_x` as input. Examples are in the [model folder](https://github.com/knodle/knodle/tree/develop/knodle/model/).
 
+If you know which denoising method you want to use, you can directly call the corresponding module (the list of currently supported methods is provided [below](https://github.com/knodle/knodle/tree/style_guide#denoising-methods)).
+
 Example for training the baseline classifier:
 
 ```python
-from knodle.model import LogisticRegressionModel
-from knodle.trainer import TrainerConfig
-from knodle.trainer.baseline.baseline import NoDenoisingTrainer
+from knodle.model.logistic_regression_model import LogisticRegressionModel
+from knodle.trainer.baseline.majority import MajorityVoteTrainer
 from knodle.data import get_imdb_dataset
+from knodle.data.transformer import get_labels
+from knodle.data.tfidf import get_tfidf_features
+
+NUM_OUTPUT_CLASSES = 2
 
-OUTPUT_CLASSES = 2
+train_df, dev_df, test_df, z_train_rule_matches, train_rule_matches_z, mapping_rules_labels_t = \
+        get_imdb_dataset()
```

Comment:
I am not sure that `get_imdb_dataset` still exists in `knodle.data`

Reply:
not, it doesn't haha :D part of a major Readme PR